integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -32,16 +33,30 @@
 writer_confs={'spark.sql.legacy.parquet.datetimeRebaseModeInWrite': 'CORRECTED',
               'spark.sql.legacy.parquet.int96RebaseModeInWrite': 'CORRECTED'}
 
+# https://github.com/rapidsai/cudf/issues/7152
```

Comment:
we could write the tests and just xfail them and we should file a followup issue to track it.

Reply:
hmm... ok. I will see if I can do that and still use our data_gens